src/main/java/com/easyrest/pages/waiterPanel/AssignedWaiterOrders.java

```diff
@@ -0,0 +1,12 @@
+package com.easyrest.pages.waiterPanel;
+
+import com.easyrest.pages.WaiterPage;
+import org.openqa.selenium.WebDriver;
+
+public class AssignedWaiterOrders extends WaiterPage {
```

Comment:
Why does this class extend waiter page? You can extend only when class use all functionality of parent class